eclass/llvm-utils.eclass

```diff
@@ -150,4 +150,21 @@ llvm_prepend_path() {
 	export PATH=${new_path[*]}
 }
 
+# @FUNCTION: llvm_cmake_usex_musl
+# @DESCRIPTION:
+# Determine whether the given LLVM project should be built with musl
+# support. That should be the case under the following ciricumstances:
+#
+# - CTARGET is a musl environment.
+# - The host profile has the `elibc_musl` flag.
+#
+# If musl should be used, echo "yes", otherwise echo "no".
+llvm_cmake_usex_musl() {
+	if [[ "${CTARGET}" == *-*-*-musl* ]]; then
+		echo "yes"
+	else
+		usex elibc_musl
```

Comment:
Sorry if that wasn't clear but I think you can remove `usex elibc_musl`, in favor of using `${CTARGET:-${CHOST}}`.